static/app/views/userFeedback/userFeedbackEmpty.tsx

```diff
@@ -44,23 +44,22 @@ export function UserFeedbackEmpty({projectIds}: Props) {
 
     if (hasAnyFeedback === false) {
       // send to reload only due to higher event volume
```

Comment:
Is this comment still relavent?

Reply:
@malwilley yes

Reply:
Ah okay I see you did make the event name null above, cool